packages/playwright-crawler/src/internals/utils/playwright-utils.ts

```diff
@@ -19,15 +19,99 @@
  * @module playwrightUtils
  */
 
+import { readFile } from 'node:fs/promises';
 import ow from 'ow';
 import { Page, Response, Route } from 'playwright';
+import { LruCache } from '@apify/datastructures';
```

Comment:
undeclared dependency

Reply:
Looking at the other packages - we probably need to go through them and update dependencies everywhere - e.g. puppeteer crawler does not have half of the packages there either, pretty sure other ones are worth checking too... But I guess not within this PR? Should I create an issue?